portality/background_helper.py

```diff
@@ -0,0 +1,48 @@
+""" collections of wrapper function for helping you to create BackgroundTask
```

Comment:
Can I suggest that we put this somewhere other than the root of the repo.  I propose in either `portality.lib` or `portality.tasks`.  I'm leaning towards the latter, but open to persuasion.

Reply:
`portality.lib` seem contain set of function that can be reuse / share to other project. but common pattern of background task could be only apply in DOAJ.
`portality.tasks` seem only contain file of `BackgroundTask`, I will move it to `portality.tasks.helpers`